src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -94,11 +95,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $encoder = $this->getContainer()->get('security.encoder_factory')->getEncoder($userClass);
         $bcryptWithoutEmptySalt = !$emptySalt && $encoder instanceof BCryptPasswordEncoder;
+        $argon2iWithoutEmptySalt = !$emptySalt && $encoder instanceof Argon2iPasswordEncoder;
```

Comment:
+1 even though I'm not a decider; I wanted to do something like that but thought it would be adding too much for the scope of this PR.

Is that something I could implement here or separate PR?